src/Couchbase.Lite.Support.Android/Couchbase.Lite.Support.Android.csproj

```diff
@@ -83,6 +83,9 @@
     <EmbeddedNativeLibrary Include="..\..\vendor\couchbase-lite-core\build_cmake\android\lib\x86\libLiteCore.so">
       <Link>x86\libLiteCore.so</Link>
     </EmbeddedNativeLibrary>
+    <EmbeddedNativeLibrary Include="..\..\vendor\couchbase-lite-core\build_cmake\android\lib\x86_64\libLiteCore.so">
+      <Link>x86_64\libLiteCore.so</Link>
+    </EmbeddedNativeLibrary>

```

Comment:
This is not part of this PR.  Let's handle it separately.

Reply:
Not sure why it's appeared here...

Reply:
Check your commit in SourceTree.  Make sure it doesn't have anything you don't recognize.  It looks like you pulled the cobalt branch into the master branch maybe?

Reply:
yes, that's what I did..